lib/team.rb

```diff
@@ -0,0 +1,38 @@
+require './lib/player'
+
+class Team
+
+  attr_reader :country,
+              :players
+
+  def initialize(country)
+    @country = country
+    @players = []
+    @eliminated = false
+  end
+
+  def add_player(player)
+    @players << player
+  end
+
+  def eliminated?
+    @eliminated
+  end
+
+  def eliminated
+    if @eliminated == false
+      @eliminated = true
+    end
+  end
+
+  def players_by_position(position)
```

Comment:
This is a perfect implementation of `.each`!  See if you can refactor this to use `.find_all` instead 😄